content/projects/oop/person/_index.md

```diff
@@ -81,5 +81,5 @@ Also, it's really important that you submit good clean code! Make sure you name
 
 Mark this as Excellent if:
 
-1. **There is a separate function just for building the interests string** If the student makes multiple small, single purpose, clear functions instead of one giant function then that is good.
-2. Makes use of templating instead of string concatenation as much as possible. In Python this means using f-strings, in Javascript it means using template literals, in Java this means using `String.format`.
\ No newline at end of file
+1. **There is a separate function just for building the interests string**. If the student makes multiple small, single purpose, clear functions instead of one giant function then that is good.
+2. Makes use of templating instead of string concatenation as much as possible. In Python this means using f-strings, in Javascript it means using template literals, in Java this means using `String.format`.
```

Comment:
Since you're already working on this file, this addition is needed. We don't want leaners thinking that any project that meets these two points should be automatically marked as excellent.

```suggestion
2. Makes use of templating instead of string concatenation as much as possible. In Python this means using f-strings, in Javascript it means using template literals, in Java this means using `String.format`.
3. Also meets normal excellence criteria as specified on Tilde under "HOW DO I CHOOSE A STATUS?". 
```